src/nwb_conversion_tools/tools/data_transfers.py

```diff
@@ -61,14 +63,143 @@ def get_globus_dataset_content_sizes(
 
     recursive_flag = " --recursive" if recursive else ""
     contents = json.loads(
-        _deploy_process(
+        deploy_process(
             command=f"globus ls -Fjson {globus_endpoint_id}:{path}{recursive_flag}", catch_output=True, timeout=timeout
         )
     )
     files_and_sizes = {item["name"]: item["size"] for item in contents["DATA"] if item["type"] == "file"}
     return files_and_sizes
 
 
+def transfer_globus_content(
+    source_id: str,
```

Comment:
would you mind this change to make it a bit more explicit?

```suggestion
    source_globus_id: str,
```